test/archethic/shared_secrets_test.exs

```diff
@@ -26,4 +28,109 @@ defmodule Archethic.SharedSecretsTest do
       assert SharedSecrets.has_origin_public_key?(pb_key)
     end
   end
+
+  describe "is_synchronized?" do
+    setup do
```

Comment:
Can't you use `ArchethicCase` which provide a lot of default mock ?

Reply:
to make test asynchronous.